pkg/storage/unified/sql/service.go

```diff
@@ -49,8 +49,9 @@ type service struct {
 
 	authenticator interceptors.Authenticator
 
-	log log.Logger
-	reg prometheus.Registerer
+	log                   log.Logger
+	reg                   prometheus.Registerer
+	unifiedStorageMetrics *resource.StorageApiMetrics
```

Comment:
Then we could write here and in all other places something like
```suggestion
	storageMetrics *resource.StorageMetrics
```